addons/account/models/account_move.py

```diff
@@ -1320,24 +1320,63 @@ def _compute_payments_widget_to_reconcile_info(self):
             pay_term_lines = move.line_ids\
                 .filtered(lambda line: line.account_id.account_type in ('asset_receivable', 'liability_payable'))
 
-            domain = [
-                ('account_id', 'in', pay_term_lines.account_id.ids),
-                ('parent_state', '=', 'posted'),
-                ('partner_id', '=', move.commercial_partner_id.id),
-                ('reconciled', '=', False),
-                '|', ('amount_residual', '!=', 0.0), ('amount_residual_currency', '!=', 0.0),
-            ]
+            def _get_common_domain():
```

Comment:
also usually inner function are placed at the top of the function, it's better for readability :smile: